django_toolkit/fallbacks/circuit_breaker/circuit_breaker.py

```diff
@@ -90,9 +90,7 @@ def _increase_failure_count(self):
 
         # Between the cache.add and cache.incr, the cache MAY expire,
         # which will lead to a circuit that will eventually open
-        self.cache.add(
-            self.rule.failure_cache_key, 0, self.failure_timeout
-        )
+        self.cache.add(self.rule.failure_cache_key, 0, self.failure_timeout)
```

Comment:
Shouldn't you create the `self.rule.request_cache_key` here, like you did with `self.rule.failure_cache_key` in `_increase_failure_count` for the same reason?

Reply:
It is not necessary. When `should_increase_request_count` is true, the class will always increment a request in the cache, so when it arrives at the` _increase_failure_count` function, the request cache has already been created

Reply:
you have a point!